bin/helpers/capabilityHelper.js

```diff
@@ -132,6 +132,14 @@ const validate = (bsConfig, args) => {
     // if parallels specified via arguments validate only arguments
     if (!Utils.isUndefined(args) && !Utils.isUndefined(args.parallels) && !Utils.isParallelValid(args.parallels)) reject(Constants.validationMessages.INVALID_PARALLELS_CONFIGURATION);
 
+    if(!Utils.isUndefined(args)){
+      if(!Utils.isUndefined(args.username) && !Utils.isUndefined(args.key))
```

Comment:
What if I don't provide "" in args and set it in browserstack.json. Please consider that case too. 
Also, can you move this validation a bit up? where we already have validation for username and password? https://github.com/browserstack/browserstack-cypress-cli/pull/118/files#diff-26dee8fe62c5e891965d2de49dd74bd0fad786ae2544889d5519d222a6145256R119